pylatex/basic.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+"""
+This module implements several classes that represent basic latex commands.
+
+..  :copyright: (c) 2014 by Jelte Fennema.
+    :license: MIT, see License for more details.
+"""
+
+from .base_classes import CommandBase, Environment, ContainerCommand
+from .package import Package
+
+
+class BasicCommand(CommandBase):
+    """A command which takes no arguments or options."""
+
+    def __init__(self):
+        super().__init__()
+
+
+class NewPage(BasicCommand):
+    """A command that adds a new page to the document."""
+
+
+class LineBreak(BasicCommand):
+    """A command that adds a line break to the document."""
+
+
+class NewLine(BasicCommand):
+    """A command that adds a new line to the document."""
+
+
+class HFill(BasicCommand):
+    """A command that fills the current line in the document."""
+
+
+class FontSize(Environment):
+    """An environment which changes the font size."""
```

Comment:
Remove this class and just have:

```python
class HugeText(Environment):
    """An environment which makes the text size 'Huge'."""

    _latex_name = "Huge"

    def __init__(self, data=None):
        """
        Args
        ----
        data : str or `~.LatexObject`
            The string or LatexObject to be formatted.
        """

        super().__init__(data=data)
```

Then have the other ones simply be:
```python
class LargeText(HugeText):
    """An environment which makes the text size 'Large'."""

    _latex_name = "Large"
```

That's much easier and a lot less code.